ott/core/segment.py

```diff
@@ -110,6 +116,64 @@ def segment_point_cloud(
   return segmented_x, segmented_a, num_segments
 
 
+def _segment_interface(
+    x: jnp.ndarray,
+    y: jnp.ndarray,
+    eval_fn: Callable[[jnp.ndarray, jnp.ndarray, jnp.ndarray, jnp.ndarray],
+                      jnp.ndarray],
+    segment_ids_x: Optional[jnp.ndarray] = None,
+    segment_ids_y: Optional[jnp.ndarray] = None,
+    num_segments: Optional[int] = None,
+    indices_are_sorted: Optional[bool] = None,
```

Comment:
This can be just `indices_are_sorted: bool = False` (based on jax docs).

Reply:
Also in other places.

Reply:
that works! will fix for now here.